leeanp.php

```diff
@@ -24,3 +24,6 @@
 // Run the plugin setup.
 $setup = new PluginSetup();
 register_activation_hook( __FILE__, [ $setup, 'maybe_deactivate' ] );
+
+// Init ECommerce endpoints.
+add_action( 'rest_api_init', [ $setup, 'init' ] );
```

Comment:
We are going to have probably 4 or 5 more endpoints at the end. Do you think it's ok if we put all the endpoints init here and delete the Bootstrap then? thank you! @mitogh